shared/teams/join-team/index.js

```diff
@@ -0,0 +1,146 @@
+// @flow
+import React from 'react'
+import {Box, Button, HeaderHoc, Icon, Input, PopupDialog, Text, ScrollView} from '../../common-adapters/index'
+import {isMobile} from '../../constants/platform'
+import {globalColors, globalMargins, globalStyles} from '../../styles'
+
+import type {Props} from './'
+
+const styleContainer = {
+  ...globalStyles.flexBoxCenter,
+  ...(isMobile ? {} : {cursor: 'default'}),
+  minHeight: 40,
+  paddingLeft: globalMargins.medium,
+  paddingRight: globalMargins.medium,
+  borderTopLeftRadius: 4,
+  borderTopRightRadius: 4,
+}
+
+const stylePadding = isMobile
+  ? {
+      paddingTop: globalMargins.xlarge,
+    }
+  : {
+      marginBottom: 80,
+      marginLeft: 80,
+      marginRight: 80,
+      marginTop: 90,
+    }
+
+const Contents = (props: Props) => (
```

Comment:
likely not needed just move ScrollView into renderContents.
Also if you're going to call a function, just make it a stateless component
rename renderContents -> RenderContents and call it

```
<ScrollView>
  <RenderContents {...props} />
</ScrollView>
```

(but really just get rid of Contents)